pkg/envoy/eds/cluster_load_assignment.go

```diff
@@ -12,21 +12,23 @@ import (
 )
 
 const (
-	zone = "zone"
+	zone                  = "zone"
+	localClusterPriority  = uint32(0)
+	remoteClusterPriority = uint32(1)
```

Comment:
What if there are multiple remote clusters, are they all given equal remote cluster priority ? Shouldn't this be determined by the location of the cluster with respect to the source ?

Reply:
In this PR:

* Yes, remote clusters are given the same priority. Any traffic outbound to the local cluster will be split based on weights. Sure we can assign priority to each remote clusters. I prefer doing in to another PR. We also need more works on how we use priority.
* `zone` field here is used as the name of the cluster. It's part of the endpoint name, and can be helpful for debugging.

Reply:
thanks for the clarification. I have one more concern here, we already have the SMI traffic split and we might want to think how that will co-exist with this